src/w3c/templates/cgbg-headers.js

```diff
@@ -4,16 +4,22 @@ import { l10n } from "./headers.js";
 import showLink from "./show-link.js";
 import showLogo from "./show-logo.js";
 import showPeople from "./show-people.js";
+import { updateSpecTitleElem } from "../../core/utils.js";
 
 export default conf => {
   const existingCopyright = document.querySelector(".copyright");
   if (existingCopyright) {
     existingCopyright.remove();
   }
+
+  const specTitleElem = updateSpecTitleElem(conf);
+  const specTitleElemClone = specTitleElem.cloneNode(true);
+  const fragment = document.createDocumentFragment();
+  fragment.append(...specTitleElemClone.childNodes);
```

Comment:
We should be able to directly insert `.childNodes` rather than creating a fragment.

Reply:
That's a great suggestion! Forgot `append()` will just adopt the nodes.